linode/linode_api4-python
# new: Add site_type to `Region`

## 📝 Description

Expose a new field `site_type` to `Region`.

## ✔️ How to Test

`tox`
`make TEST_CASE=test_get_regions testint`

Manual Test:
1. In a sandbox environment, run the following:
```python
import os

from linode_api4 import LinodeClient

client = LinodeClient(os.getenv("LINODE_TOKEN"))

regions = client.regions()

if len(regions) > 0:
    print(f"Region {regions[0].id} site type: {regions[0].site_type}")
```
2. Observe that region site type is in the output.
